### DISCUSSION TITLE
Cad Export

### DISCUSSION THREAD

**Discussion by eykaraduman** [FEALiTE/FEALiTE2D#9] [Ideas]
Hi,
I think, when exporting to cad, a selectable output file path and dxf support option will be good. For dxf support [netDxf](https://github.com/haplokuon/netDxf) can be used.

**Comment by SaMohamed** [comment]
Thank you for your suggestion, I will try to do a more export options like:
 1- dxf export. 
 2- plotting element diagrams using oxyplot library.
But this will increase the library dependences so I think export option should be in a separate library but in the same project.
So what do you think?

**Reply by eykaraduman** [reply]
OxyPlot will be a good alternative with a searchable moment, sherar values at desired point of beam.
